src/holochainClient.js

```diff
@@ -1,9 +1,10 @@
 import { connect as hcWebClientConnect } from '@holochain/hc-web-client'
 import { get } from 'lodash/fp'
 import mockCallZome from 'mock-dnas/mockCallZome'
+import { findInstanceForAgent } from 'utils/conductorConfig'
```

Comment:
@joel this is the file that uses `findInstanceForAgent`, see `conductorConfig` as well